### DISCUSSION TITLE
Error running ssdp

### DISCUSSION THREAD

**Discussion by prvsousa** [codingjoe/ssdp#92] [Q&A]
Hi,

I got this error when trying to run this project:

```➔ python3 test_ssdp.py 
Traceback (most recent call last):
  File "test_ssdp.py", line 7, in <module>
    from . import fixtures
ImportError: cannot import name 'fixtures' from '__main__' (test_ssdp.py)

```

How can I solve this?

**Comment by codingjoe** [comment]
Bom dia @patricia93sousa!

thanks for reaching out. I am not sure how you ran into this error. Can you maybe give me a little more information about that you have been toding to get this error message?

I am assuming you are building some kind of IoT application?

Best
-Joe
